lib/generators/sufia/templates/workflow.json.erb

```diff
@@ -9,19 +9,45 @@
                     "name": "deposit",
                     "from_states": [],
```

Comment:
@mjgiarlo @jcoyne @jeremyf I wonder if leveraging the `work.depositor` would be adequate for this type of feature. Let's say, when the depositor of a work is on `#show` or `#edit`, they are exposed the necessary bits of UI that aids in the awareness of a required review and submitting the work for approval?